### DISCUSSION TITLE
Chapter-wise export

### DISCUSSION THREAD

**Discussion by florian-berger** [audiamus/BookLibConnect#44] [Ideas]
Hey,

first of all, a big thanks for this project. I searched a lot for the possibility to download Audible books as mp3, until I found this repo.

But I'm missing the functionality to export audio books in a chapter-wise format. If I take, for example, Harry Potter and the Order of the Phoenix, I can just download it 
- as single file, or
- splitted into 3 parts

The book itself contains around 40 chapters - the information about them is available in Audible app. Wouldn't it be great, if this BookLibConnect application is able to split the .aax file after decryption in multiple m4b files, one for each chapter?

**Comment by audiamus** [comment]
Have a look at [AAX Audio Converter](https://github.com/audiamus/AaxAudioConverter). It has all that and more. The export functionality of Book Lib Connect is providing the input data.

**Reply by florian-berger** [reply]
Perfect, thank you 👍 

I already downloaded the software, but was confused that it asks for a code I was unable to get. But it make sense that I can load the decrypted m4b there. I tested it, it works. Thank you again ❤️
